### ISSUE TITLE
How do i get this file to a website

### ISSUE THREAD

**Issue by zarzwour** [Tyrrrz/DiscordChatExporter#34]
I want to export this file to my modpack website please help

**Comment by Tyrrrz** [comment]
Can you rephrase the question?

**Comment by zarzwour** [comment]
i want the file:///C:/Users/gregg/AppData/Local/Temp/Rar$EXa21944.33471/W-Technology%20(being%20revived)%20-%20general.html to be exported to my webpage

**Comment by Tyrrrz** [comment]
You can do that manually. Uploading files is not in scope of this project.

**Comment by zarzwour** [comment]
how do i access that file?

**Comment by Tyrrrz** [comment]
Normally, by opening it.

**Comment by zarzwour** [comment]
where is it stored? i am super confused

**Comment by Tyrrrz** [comment]
You choose where to save it. It saves in the same directory as the executable by default. Judging by your URL you never extracted the zip contents.

**Comment by infominer33** [comment]
Publishing Discord Server Archives via Minimal Mistakes

https://didecentral.com/contributors-guide/discord-archive-howto/

I added a link to the wiki, and intend to create more content along these lines.

**Comment by Tyrrrz** [comment]
It looks pretty awesome @infominer33
